zipline/data/us_equity_pricing.py

```diff
@@ -43,9 +43,14 @@
     read_csv,
     read_sql,
     to_datetime,
-    Timestamp,
+    Timestamp,    
```

Comment:
May want to clean that trailing whitespace. Don't mind me, just looking for a solution to the xlocale.h issue
on another distro and package.